### PR TITLE
Sensor test

### PR DESCRIPTION
**Pull request by mmphego**

### CODE REVIEW

File: mkat_fpga_tests/test_cbf.py [hunk 1]
```diff
@@ -480,9 +480,9 @@ def get_actual_phases():
                 # correlator_fixture.katcp_rct.req.delays time.time+somethign
                 # See page 22 on ICD ?delays on CBF-CAM ICD
                 reply = correlator_fixture.katcp_rct.req.input_labels()
```
Comment (brickZA) [inline mkat_fpga_tests/test_cbf.py]: You should probably change this to be reply, informs = correl......, then you can also skip the [0] you added below.

File: mkat_fpga_tests/test_cbf.py [hunk 2]
```diff
@@ -607,63 +607,66 @@ def test_sensor_values(self):
...
-        rct.start()
-        rct.until_synced()
-
+        sensors_req = correlator_fixture.rct.req
+        array_sensors_req = correlator_fixture.katcp_rct.req
         # 1. Request a list of available sensors using KATCP command
         # 2. Confirm the CBF replies with a number of sensor-list inform messages
-        LOGGER.info (rct.req.sensor_list())
+        LOGGER.info (sensors_req.sensor_list())
+        LOGGER.info (array_sensors_req.sensor_list())
```
Comment (brickZA) [inline mkat_fpga_tests/test_cbf.py]: Don't do the request twice, just log list_reply, list_infomrs below.

File: mkat_fpga_tests/test_cbf.py [hunk 3]
```diff
@@ -479,10 +479,10 @@ def get_actual_phases():
                 # use correlator_fixture.corr_conf[]
                 # correlator_fixture.katcp_rct.req.delays time.time+somethign
                 # See page 22 on ICD ?delays on CBF-CAM ICD
-                reply = correlator_fixture.katcp_rct.req.input_labels()
-                source_name = reply.arguments[1].split()
+                reply, informs = correlator_fixture.katcp_rct.req.input_labels()
+                source_name = informs.arguments[1].split()
```
Comment (brickZA) [inline mkat_fpga_tests/test_cbf.py]: informs is going to be a list of informs, so you need to do informs[0].arguments[1].....?

Comment (mmphego) [inline mkat_fpga_tests/test_cbf.py]: reply - status is concatenated with the messages received. 
informs returns an empty list.

File: mkat_fpga_tests/test_cbf.py (hunk 1 above)
Comment (mmphego) [inline mkat_fpga_tests/test_cbf.py]: @brickZA Changes made

File: mkat_fpga_tests/test_cbf.py (hunk 2 above)
Comment (mmphego) [inline mkat_fpga_tests/test_cbf.py]: @brickZA  Done

**Pull request merged**